python/eks/cluster/requirements.txt

```diff
@@ -0,0 +1 @@
+aws-cdk.aws-eks
```

Comment:
Can you please update the requirements to include all of the used AWS libraries? Namely:     `aws_autoscaling`, `aws_ec2`, `aws_eks`, `aws_iam`, `core` and `custom_resources`.

Reply:
even if aws-eks already sets these ones as dependencies?

Reply:
I would say so. If, in a future version, `aws_eks` developers manage to remove its dependency on `aws_iam`, your application will suddenly stop working because this indirect dependency will no longer be installed. The best practice here would be to include all direct dependencies of your code in the requirements file.

Reply:
have a check please

Reply:
Looks good!